src/Presentation/Detection/Fetching/Resolver.cs

```diff
@@ -1,16 +1,20 @@
 namespace Presentation.Detection.Fetching
 {
-    using System;
     using System.Collections.Generic;
     using Business.Detection.Common.Models;
+    using Business.Detection.Fetching.Commands;
     using global::GraphQL;
 
     public class Resolver : IResolver
     {
+        private readonly IGetDetection _getDetection;
+
+        public Resolver(IGetDetection getDetection)
+        {
+            _getDetection = getDetection;
+        }
```

Comment:
Should we convert this constructor into a one liner?

Reply:
Done